[2024-08-28T08:24:46.079+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-28T08:24:46.106+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: country_data_pipeline.load_data manual__2024-08-28T08:18:44.454379+00:00 [queued]>
[2024-08-28T08:24:46.125+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: country_data_pipeline.load_data manual__2024-08-28T08:18:44.454379+00:00 [queued]>
[2024-08-28T08:24:46.126+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2024-08-28T08:24:46.176+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): load_data> on 2024-08-28 08:18:44.454379+00:00
[2024-08-28T08:24:46.184+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=439) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-28T08:24:46.185+0000] {standard_task_runner.py:72} INFO - Started process 448 to run task
[2024-08-28T08:24:46.184+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'country_data_pipeline', 'load_data', 'manual__2024-08-28T08:18:44.454379+00:00', '--job-id', '122', '--raw', '--subdir', 'DAGS_FOLDER/welcome_dags.py', '--cfg-path', '/tmp/tmpbpfqjt4u']
[2024-08-28T08:24:46.186+0000] {standard_task_runner.py:105} INFO - Job 122: Subtask load_data
[2024-08-28T08:24:46.294+0000] {task_command.py:467} INFO - Running <TaskInstance: country_data_pipeline.load_data manual__2024-08-28T08:18:44.454379+00:00 [running]> on host c63ae693b587
[2024-08-28T08:24:46.473+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='country_data_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-28T08:18:44.454379+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-28T08:18:44.454379+00:00'
[2024-08-28T08:24:46.475+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-28T08:24:47.338+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-28T08:24:47.352+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-28T08:24:47.352+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=country_data_pipeline, task_id=load_data, run_id=manual__2024-08-28T08:18:44.454379+00:00, execution_date=20240828T081844, start_date=20240828T082446, end_date=20240828T082447
[2024-08-28T08:24:47.406+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-28T08:24:47.431+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-28T08:24:47.433+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
